test/cppsuite/tests/poc.cxx

```diff
@@ -152,10 +152,10 @@ main(int argc, char *argv[])
         /* Check if default configuration should be used. */
         if (cfg.empty() && filename.empty())
             cfg = parse_configuration_from_file(
-              "../../../test/cppsuite/configurations/" + poc_test::test::default_config);
+              "configs/" + poc_test::test::default_config);
         else if (!filename.empty())
             cfg =
-              parse_configuration_from_file("../../../test/cppsuite/configurations/" + filename);
+              parse_configuration_from_file(filename);
```

Comment:
If someone is specifying a file I don't think we should assume that file is under our configs/ directory, which is why I made this change.

Reply:
Agreed